test/bigint_test.dart

```diff
@@ -0,0 +1,36 @@
+//
+// bigint_test.dart
+// Test of very wide bitvector comparison failure
+//
+//
+
+import 'dart:math';
+import 'package:rohd/rohd.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('crash compare', () {
```

Comment:
Good catch that this affects both `_doMath` and `_doCompare`, but don't these tests only test agains the `_doCompare` bug?